amy/trainings/forms.py

```diff
@@ -129,8 +136,10 @@ class BulkAddTrainingProgressForm(forms.ModelForm):
         # the template where this form is used
         "requirement",
         "state",
+        "involvement_type",
         "event",
         "url",
+        "date",
```

Comment:
Is there a widget for user when they click on the "date of occurrence" field?

Reply:
If not, then at least there should be a format specified in `help_text` for the field.

Reply:
Yes, there's a widget. I think this is used by default with DateFields. It's still possible to type in a date instead, so good idea to include the required format in the help text.